.circleci/config.yml

```diff
@@ -1,120 +1,116 @@
 version: 2.1
 
-# add your orb below, to be used in integration tests (note: a @dev:alpha
-# release must exist; you'll need to publish manually at least once)
-# you can use scripts/publish-alpha.sh to publish a @dev:alpha release
-orbs:
-  codacy: codacy/base@dev:alpha
-  orb-tools: circleci/orb-tools@8.5.0
+references:
+  git_job: &git_job
+    docker:
+      - image: codacy/git-version:1.0.8
+    working_directory: ~/workdir
+
+  circleci_job: &circleci_job
+    docker:
+      - image: circleci/circleci-cli:0.1.5705
+    working_directory: ~/workdir
+
+jobs:
+  checkout_and_version:
+    <<: *git_job
+    steps:
+      - checkout
+      - run:
+          name: Set version
+          command: |
+            /bin/git-version > .version
+      - run:
+          name: Current version
+          command: cat .version
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - workdir
+
+  tag_version:
+    <<: *git_job
+    steps:
+      - attach_workspace:
+          at: ~/
+      - add_ssh_keys:
+          fingerprints:
+          - "df:83:d7:c7:d5:79:06:c2:3b:d1:fd:e2:a3:d1:12:c5"
+      - run:
+          name: add bitbucket known host
+          command: |
+            mkdir -p ~/.ssh
+            touch ~/.ssh/known_hosts
+            ssh-keyscan github.com >> ~/.ssh/known_hosts
+      - deploy:
+          name: Tag git repository
+          command: git tag $(cat .version) && git push --tags
+
+  pack_and_validate:
+    <<: *circleci_job
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Pack orb
+          command: circleci config pack src > orb.yml 
```

Comment:
just to be extra-safe I would `rm -f orbl.yml` before

Reply:
because i'm using a single > the content of the file (if it exists) is overwritten

Reply:
you are right

Reply:
correct, np!